### ISSUE TITLE
Investigate Eye of Komodo collection problems

### ISSUE THREAD

**Issue by NutellaLicka** [TokelPlatform/token-explorer#8]
Some of the 'Eye of Komodo' collection does not identify as being in the collection. It seems like the arbitrary data has problems.

![](https://i.gyazo.com/d273c222c6ffc8f0e1b8413b8e8ec346.png)

![](https://i.gyazo.com/0f089d105e5c03429098e90371f2783a.png)

**Comment by lenilsonjr** [comment]
Not sure there's anything we can do here, only if it's some parsing issue of chain data. If it was inserted on chain with malformed data, we'd have to add a manual workaround which isn't desirable.

**Comment by piggydoughnut** [comment]
Yeah that is awkward, I can theoretically hack around it. @NutellaLicka could you ask the creator how he creates his arbitrary data? Maybe he uses some weird generator and it adds a weirdly formatted space or new line at the end 🤷🏽‍♀️ 

The problem with this is that we cannot hack all the users mistakes.

We might try and help prevent this in the future somehow, maybe scanning for weird characters? Or maybe giving them an option to choose format, e.g. JSON and then check if it is valid. That would probably help a lot.

**Comment by piggydoughnut** [comment]
@NutellaLicka did you get any information from the creator of this collection ?
